src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs/launch_analysis.cljs

```diff
@@ -43,6 +42,7 @@
        :else
        [:div {}
         [:div {} "Estimated wait time: " (.humanize (js/moment.duration (queue-status "estimatedQueueTimeMS")))]
+        [:div {} (str "There are " (queue-status "workflowsBeforeNextUserWorkflow") " workflows ahead of yours in the queue.")]
```

Comment:
Since we're completely pulling apart `queue-status`, it might be nice to destructure it in the `let`.  You could add this to the let bindings:

    {:strs [estimatedQueueTimeMS workflowsBeforeNextUserWorkflow workflowCountsByStatus]} queue-status

and then use those as variables in those 3 places.  Or maybe that's worse, I dunno.  Up to you.